.rubocop_todo.yml

```diff
@@ -1,22 +1,33 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2019-11-09 15:45:44 -0500 using RuboCop version 0.75.1.
+# on 2020-04-05 14:47:50 -0400 using RuboCop version 0.78.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+# Configuration parameters: Include.
+# Include: **/*.gemspec
+Gemspec/RequiredRubyVersion:
+  Exclude:
+    - 'skunk.gemspec'
+
 # Offense count: 1
 # Cop supports --auto-correct.
 Layout/ClosingHeredocIndentation:
   Exclude:
     - 'lib/skunk/cli/commands/status_reporter.rb'
 
+Layout/IndentHeredoc:
```

Comment:
This should be deleted?

Reply:
@bronzdoc If I delete it Rubocop complains a code style violation. I've followed their suggestion and renamed it as `Layout/HeredocIndentation`